Pycloud Deployer/infra_classes.py

```diff
@@ -0,0 +1,288 @@
+from azure.mgmt.resource import ResourceManagementClient
+from azure.mgmt.network import NetworkManagementClient
+from azure.mgmt.compute import ComputeManagementClient
+from modules import (
+    create_dbserver_parameters, 
+    create_publicip_parameters, 
+    create_nic_parameters, 
+    create_webserver_parameters, 
+    run_command_parameters
+)
+
+class WebServer():
```

Comment:
Since The DBServer and LoadBalancer classes inherit from WebServer maybe consider a more generic name "Server"?